services/npm/npm-dependency-version.service.js

```diff
@@ -14,89 +15,55 @@ export default class NpmDependencyVersion extends NpmBase {
     queryParamSchema,
   }
 
-  static examples = [
-    {
-      title: 'npm peer dependency version',
-      pattern: ':packageName/peer/:dependency',
-      namedParams: {
-        packageName: 'react-boxplot',
-        dependency: 'prop-types',
-      },
-      staticPreview: this.render({
-        dependency: 'prop-types',
-        range: '^15.5.4',
-      }),
-      keywords,
-    },
-    {
-      title: 'npm peer dependency version (scoped)',
-      pattern: ':scope?/:packageName/peer/:dependencyScope?/:dependency',
-      namedParams: {
-        scope: '@swellaby',
-        packageName: 'eslint-config',
-        dependency: 'eslint',
-      },
-      staticPreview: this.render({
-        dependency: 'eslint',
-        range: '^3.0.0',
-      }),
-      keywords,
-    },
-    {
-      title: 'npm dev dependency version',
-      pattern: ':packageName/dev/:dependency',
-      namedParams: {
-        packageName: 'react-boxplot',
-        dependency: 'eslint-config-standard',
+  static openApi = {
+    '/npm/dependency-version/{packageName}/{dependency}': {
+      get: {
+        summary: 'NPM (prod) Dependency Version',
+        parameters: [
+          pathParam({
+            name: 'packageName',
+            example: 'react-boxplot',
+            description: packageNameDescription,
+          }),
+          pathParam({
+            name: 'dependency',
+            example: 'simple-statistics',
+            description: packageNameDescription,
+          }),
```

Comment:
I like the fact that we've simplified this into two fields with an explanation of what this scoping business is. I'm personally somewhat familiar with the "scope" terminology because I've occasionally come across it in the Shields.io code, but I feel that someone which isn't a JS power-user may not be familiar with the wording.

However, I must admit that I was briefly confused both when reading the code and trying the badge out in the review app. I think this boils down to the fact that `package` is a package and `dependency` is also a package, and we're basically overloading the word package in the path parameter name, the explanation of the parameters, and the examples within the explanations.

I think it may help to replace any mentions of package by dependency in the dependency parameter description (even if a dependency is technically also a package), and instead of giving `package` and `@author/package` as examples in the description, use something more specific-sounding like  `my-package` and `@author/my-package`